### DISCUSSION TITLE
zh/docs/provider/oauth/DingTalk/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2191] [Docs comments]
# zh/docs/provider/oauth/DingTalk/

添加钉钉 OAuth 到您的应用程序

https://casdoor.org/zh/docs/provider/oauth/DingTalk/

**Comment by qiaoanqiao** [comment]
Client ID和Client secret与钉钉后台对应不上该咋填

**Reply by hsluoyz** [reply]
The docs has been updated

**Comment by yangsijie666** [comment]
可以给详细些的文档吗？包括钉钉后台的配置，casdoor 应用、组织相关的配置。目前整个文档太零散了，相关资料也很少

**Reply by hsluoyz** [reply]
The docs has been updated

**Comment by sqkkyzx** [comment]
显示“不合法的参数unionid”

**Reply by aozima** [reply]
遇到同样的问题，后面有找到原因和解决方法吗？

**Reply by aozima** [reply]
后面通过调试，发现是 `/v1.0/contact/users/me` 这个API接口没权限，但这个接口的错误处理没能输出报错信息。在钉钉里面把 Contact.User.Read 权限打开即可。

**Comment by explorer-exe-bilibili** [comment]
成员信息读权限也要开，要不然会报错
